test-case/check-fw-echo-reference.sh

```diff
@@ -84,17 +75,20 @@ do
         fi
         # run echo reference test
         dlogc "alsabat -P $pb_dev --standalone -c $channel -f $fmt -r $rate -n $frames -F $frequency"
-        alsabat -P "$pb_dev" --standalone -c "$channel" -f "$fmt" -r "$rate" -n "$frames" -F "$frequency" &
+        alsabat -P "$pb_dev" --standalone -c "$channel" -f "$fmt" -r "$rate" -n "$frames" -F "$frequency" & alsabatPID=$!
         # playback may have low latency, add 0.5 second delay to aviod recording zero at beginning.
         sleep 1
```

Comment:
The comment says 0.5 but the code says 1.
Did you mean "high" latency?

Reply:
it should be 1 second delay.

Reply:
updated.